### PR TITLE
WIP: improving docs flow

### PR DESCRIPTION
**Pull request by bethaitman-da**
Tidying up the table of contents a bit, including:

- removing unnecessary stdlib wrapper page
- changing the main toc to sentence case
- improving page titles to make the toc read better (ie not say `DAML` over and over and over)
- shifting a few pages around in the toc (but not moving the source file) to tidy it up

### CODE REVIEW

File: daml-foundations/daml-ghc/base-rst-prefix.rst [hunk 1]
```diff
@@ -3,8 +3,25 @@
...
-Base for DAML 1.2
-=================
+The standard library
+====================
 
-The base domain of the DAML standard library contains various modules with
+The DAML standard library is a collection of DAML modules organised in domains
+that can be used to implement concrete applications.
+
+This page covers the Base domain, which contains various modules with
```
Comment (cocreature) [inline daml-foundations/daml-ghc/base-rst-prefix.rst]: It isn’t quite clear to me whether “Base domain” is intended to be a technical term in the sense that “Module” and “Package” are and what exactly it encompasses.

**Comment by bethaitman** [comment]
Mainly I never got round to finishing it. Thanks for the reminder, I'll try to pick this up again.

File: daml-foundations/daml-ghc/base-rst-prefix.rst (hunk 1 above)
Comment (bethaitman) [reply daml-foundations/daml-ghc/base-rst-prefix.rst]: I think I blindly copied this from somewhere, and it doesn't make sense - I'll remove it.

**Comment by bethaitman** [comment]
I remember why this got stuck in draft - I wanted to add the introduction to DAML page. That's a slightly bigger blocker. I wonder if I can redo this PR without it...

**Pull request closed without merging**